api_app/visualizers_manager/visualizers/domain_reputation_services.py

```diff
@@ -0,0 +1,184 @@
+from logging import getLogger
+from typing import Dict, List
+
+from api_app.core.choices import Status
+from api_app.visualizers_manager.classes import Visualizer
+from api_app.visualizers_manager.enums import VisualizableIcon
+
+logger = getLogger(__name__)
+
+
+class DomainReputationServices(Visualizer):
+    def run(self) -> List[Dict]:
+        first_level_elements = []
+        second_level_elements = []
+        third_level_elements = []
+
+        for analyzer_report in self.analyzer_reports():
+            if (
+                "Malicious_Detector" in analyzer_report.config.name
+                or analyzer_report.config.name == "GoogleSafebrowsing"
+            ):
+                printable_analyzer_name = analyzer_report.config.name.replace("_", " ")
+                logger.debug(f"{printable_analyzer_name=}")
+                logger.debug(f"{analyzer_report.config.python_complete_path=}")
+                logger.debug(f"{analyzer_report=}")
+                third_level_elements.append(
+                    self.Bool(
+                        name=printable_analyzer_name,
+                        value=analyzer_report.report["malicious"],
+                    )
+                )
+
+        analyzer_report = self.analyzer_reports().get(
+            config__name="VirusTotal_v3_Get_Observable"
+        )
+        if analyzer_report:
```

Comment:
This will crash if the report does not exist.
The `if` condition should be converted to a `try except AnalyzerConfig.DoesNotExists`